monitor/stat_retriever.go

```diff
@@ -2,6 +2,10 @@ package monitor
 
 // this code cut and pasted from the promethious repo
 
+// this code cut and pasted from the promethious repo
+
+// this code cut and pasted from the promethious repo
+
```

Comment:
![image](https://user-images.githubusercontent.com/85355/36126461-23fc91c8-108b-11e8-82f4-cc7f9d198e27.png)

Reply:
I had removed it.... damn I must of reversed it by mistake :-)